controller/internal/routes/version_router.go

```diff
@@ -53,14 +54,23 @@ func (ir *VersionRouter) Register(ae *env.AppEnv) {
 }
 
 func (ir *VersionRouter) List(ae *env.AppEnv, rc *response.RequestContext) {
-
 	buildInfo := build.GetBuildInfo()
 	data := rest_model.Version{
 		BuildDate:      buildInfo.GetBuildDate(),
 		Revision:       buildInfo.GetRevision(),
 		RuntimeVersion: runtime.Version(),
 		Version:        buildInfo.GetVersion(),
+		APIVersions: []*rest_model.APIVersion{
```

Comment:
add unversioned paths/v0 for consistency?

Reply:
There is no v0. There is just v1 as we have it right now. It just happens to be temporarily available w/ and w/o path prefix.

The one w/o the path prefix is going to go away - it is deprecated. I don't see a reason to add it.